chapter11/red/Date/Oogaboogoo.pm

```diff
@@ -0,0 +1,80 @@
+package Date::Oogaboogoo;
+
+=head1 NAME
+
+Date::Oogaboogoo
+
+=head1 VERSION
+
+Version 1.0
+
+=head1 SYNOPSIS
+
+  use Date::Oogaboogo;
+
+  my $day = Date::Oogaboogoo::number_to_day_name(1); # dip
+  my $mon = Date::Oogaboogoo::number_to_month_name(3); # rod
+
+=head1 DESCRIPTION
+
+A date formatting library for display of dates in the Oogaboogoo calendar.
+
+=cut
+
+use strict;
+use warnings;
+require Exporter;
+use vars qw( @ISA @EXPORT_OK );
+
+@ISA = 'Exporter';
```

Comment:
Nah, I assigned a 1 element list to an array. From [perldoc](http://perldoc.perl.org/perldata.html#List-value-constructors): _List values are denoted by separating individual values by commas (and enclosing the list in parentheses where precedence requires it)_.